.github/workflows/continuous-integration.yaml

```diff
@@ -29,9 +29,6 @@ jobs:
   lint:
     name: Code standards (linting)
     runs-on: ubuntu-20.04
-    strategy:
-      matrix:
-        workdir: [".", "./cmd/otel-allocator", "./cmd/operator-opamp-bridge"]
```

Comment:
why was this removed? 

IIRC this was assuming the lint runs per all subprojects

Reply:
The lint target in our Makefile already does this, so it seems redundant unless we want to have each subproject check show up as a separate CI job.